code/obj/submachine/weapon_vendor.dm

```diff
@@ -409,6 +413,12 @@
 	path = /obj/item/device/prisoner_scanner
 	description = "A device used to scan in prisoners and update their security records."
 
+/datum/materiel/utility/sectowel
+	name = "Security Towel"
+	path = /obj/item/cloth/towel/security
+	description = "A Towel used to clean off the clow- criminal blood!"
```

Comment:
I know it's minor, but don't capitalize towel here.

Reply:
That made me chuckle. I was speaking of the description.

Reply:
>:(